### DISCUSSION TITLE
How to update dependencies with N-1 version.

### DISCUSSION THREAD

**Discussion by thangarajav** [renovatebot/renovate#25414] [Request Help] [answered]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

Hi Team,

How to update the dependencies with N-1 version, if say latest version is 1.2.1 and next lower version is 1.2.0.  I don't want to go with latest version, instead i would like to go with N-1 version (1.2.0). Is that possible to configure the renovatebot to update my dependency with N-1 version. 
Also is that possible to configure the renovatebot with the dependency version which I like to update and not the latest version availabl.

### What you tried so far.

Not tried yet, like to discuss before proceeding 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Please ask one question per topic. It's not possible to automatically have N-1. I don't recommend this either for packages which are quite stable (e.g. react, lodash, etc)

**Comment by thangarajav** [comment]
Thank you,

1. If say I have a list of stable version, How can I mention to renovatebot to update the that stable version which I prefer. 
2. Whether renovatebot will update the dependencies for P2 and P3 Components also.
